src/views/MaterialTopTabBar.js

```diff
@@ -140,11 +140,21 @@ export default class TabBarTop extends React.PureComponent<Props> {
     /* eslint-disable no-unused-vars */
     const { navigation, renderIcon, getLabelText, ...rest } = this.props;
 
+    // Set route.accessibilityLabel that TabBar will reference internally
+    const navigationState = navigation.state;
+    navigationState.routes.forEach((route, idx, routes) => {
+      route.index = idx;
+      route.routesLength = routes.length;
+      route.accessibilityLabel = this.props.getAccessibilityLabel({
```

Comment:
What does this do?

Reply:
@satya164 Quite embarrassing -- mistakenly referenced an old fork of react-native-tab-view that used `route.accessibilityLabel` 😓